code/style.css

```diff
@@ -5,3 +5,326 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+body{
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color:rgb(136, 177, 177);
+}
+header{
+  width: 100%;
+}
+img{
+  object-fit: cover;
+  width: 100%;
+  height: auto;
+  top: 0px;
+}
+
+h1{
+  font-size: 30px;
+  color: white;
+  z-index: 1;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 100;
+  margin: 0;
+}
+
+h2{
+  font-size: 20px;
+  color: white;
+  z-index: 1;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 200;
+  margin: 0;
+}
+h3{
+  font-size: 20px;
+  font-weight: 200;
+  text-align: center;
+  color: white;
+}
+h4{
+  padding-left: 25px;
+  font-size: 17px;
+  font-weight: 200;
+  float: left;
+  color: white;
+}
+p{
+  margin: 0px;
+  padding: 10px;
+  font-size: 13px;
+  color: white;
+}
+.p-padding-none{
+  padding: 0;
+}
+
+.headline-text{
+  display: flex;
+  flex-direction: column;
+  position:absolute;
+  width: 100%;
+  height: auto;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  z-index: 1;
+  top: 100px;
+}
+#section1, #section3{
+  background-color: #356B71;
+}
+
+form{
+  max-width: 100%;
+  padding: 10px;
+}
+.form-header{
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color:lightslategray;
+  border-radius: 5px 5px 0px 0px;
+  padding-bottom: 10px;
+  background-color: #577e83;
+  border-bottom: 1px solid white;;
+}
+
+.form-wrapper{
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #578378;
+  align-items: center;
+  width: 95%;
+  border-radius: 5px;
+  position: static;
+  margin: 30px 0px 30px 0px;
+  box-shadow: 5px 5px 5px #7A8C8E;
+}
+label{
+ margin: 10px 10px 0px 0px;
+ font-size: 15px;
+ font-weight: 300;
+ color: white;
+}
+input{
+  margin: 5px;
+  width: auto;
+  padding: 5px;
+  border-radius: 5px;
+  border: none;
+
+}
+input:checked{
+  box-shadow: 3px 3px 3px #356B71;
+}
+input:focus{
+  background-color: honeydew;
```

Comment:
Your codes are so clear and well-written! I can read it so well: what is the function and how it will be triggered!